scholia/app/templates/pathway.html

```diff
@@ -92,11 +92,12 @@
 
 // Extensible query for muliple, optional values
 const optionalDataValuesSparql =`
-SELECT ?wpid ?organism ?organismLabel
+SELECT ?wpid ?organism ?organismLabel ?description
 WHERE {
   OPTIONAL {
     wd:{{ q }} wdt:P2410 ?wpid ;
     wdt:P703 ?organism ;
+    schema:description ?description .
```

Comment:
This will result in multiple values for each language AFAIU.

Reply:
What about `VALUES ?pathway { wd:{{ q }} }` and `?pathwayDescription`?

Reply:
Nice! That works better. Change pushed. Should be part of this PR now.